widget/list.go

```diff
@@ -93,6 +93,26 @@ func (l *List) MinSize() fyne.Size {
 	return l.BaseWidget.MinSize()
 }
 
+// RefreshItem refreshes a single item, specified by the item ID passed in.
+//
+// Since 2.4
```

Comment:
The colon needs to be included for it to be parsed correctly on our developer website:
```suggestion
// Since: 2.4
```